ANSWERS.md

```diff
@@ -1,4 +1,39 @@
 - [ ] Why would you use class component over function components (removing hooks from the question)?
+     
+      The reason of using class component over function component, is that class component allows you 
+      to use a variety of component lifecycle methods (render, componentDidMount, componentDidUpdate), that have been introduce by React team to give you the control of your components/functions.
+
```

Comment:
For class components, lifecycle methods are more pronounced, but with the `useEffect` hooks, you can target when a component mounts, when it updates, and even when it unmounts, check [here](https://reactjs.org/docs/hooks-effect.html) for details